elements/pl-latex-matrix-display/pl-latex-matrix-display.py

```diff
@@ -0,0 +1,56 @@
+import prairielearn as pl
+import lxml.html
+import numpy as np
+import chevron
+
+
+def prepare(element_html, element_index, data):
+    element = lxml.html.fragment_fromstring(element_html)
+    required_attribs = ['params-name']
+    optional_attribs = ['label', 'digits', 'presentation-type']
+    pl.check_attribs(element, required_attribs, optional_attribs)
+
+
+def render(element_html, element_index, data):
+    element = lxml.html.fragment_fromstring(element_html)
+
+    # Get the number of digits to output
+    digits = pl.get_integer_attrib(element, 'digits', 2)
+    # Get the presentation type
+    presentation_type = pl.get_string_attrib(element, 'presentation-type', 'f')
+
+    var_name = pl.get_string_attrib(element, 'params-name')
+    # Get value of variable, raising exception if variable does not exist
+    var_data = data['params'].get(var_name, None)
+    if var_data is None:
+        raise Exception('No value in data["params"] for variable %s in pl-latex-matrix-display element' % var_name)
+
+    # If the variable is in a format generated by pl.to_json, convert it
+    # back to a standard type (otherwise, do nothing)
+    var_data = pl.from_json(var_data)
+
+    if np.isscalar(var_data):
+        raise Exception('Value in data["params"] for variable %s in pl-latex-matrix-display element must be an 1D or 2D array' % var_name)
+    else:
+        # Wrap the variable in an ndarray (if it's already one, this does nothing)
+        var_data = np.array(var_data)
+        # Check shape of variable
+        if var_data.ndim > 2:
```

Comment:
Maybe check that it is exactly a 2d array? (So, disallow 1d arrays?) This would be consistent with `pl-matrix-output`.